grpc_server.go

```diff
@@ -90,6 +90,16 @@ func (s *GRPCServer) Init() error {
 	return nil
 }
 
+// Stop calls Stop on the underlying grpc.Server
+func (s *GRPCServer) Stop() {
```

Comment:
Any reason to not just call this `Close() error` to implement `io.Closer`? (This or graceful)

Reply:
I thought about `Close`, but ended up using `Stop` because that's what the grpc server calls it, and to keep the similarity between `Stop` and `StopGraceful`. 

I don't think fulfilling `io.Closer` is a string reason, since it doesn't fulfill any other `io` interfaces, but I'm not opposed to naming it differently.